code/modules/organs/internal/species/machine/wireless_access.dm

```diff
@@ -0,0 +1,83 @@
+/obj/item/organ/internal/machine/wireless_access
+	name = "wireless access point"
+	desc = "A module that allows a Bishop frame to wirelessly access machines that it is authorized to interact with."
+	organ_tag = BP_WIRELESS_ACCESS
+	parent_organ = BP_HEAD
+
+	action_button_name = "Access Internal Computer"
+
+	/// This frame's internal PDA.
+	var/obj/item/modular_computer/handheld/pda/synthetic_internal/internal_pda
```

Comment:
Should the wireless access function be de-coupled from a internal PDA function?
- I´d assume a internal PDA to be much more common than a wireless access point to access various machinery.

-> If decoupled, there should also be a way to insert the ID card into the internal PDA (so you can access programs that require a inserted id)
-> If the ability to insert the ID into the internal PDA is added, the ability to (forcefully) remove it should also be added

Reply:
will put it on the to-do list, yeah